WordPress/Classes/Services/ReaderTopicService+FollowedInterests.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+// MARK: - ReaderFollowedInterestsService
+
+/// Protocol representing a service that retrieves the users followed interests/tags
+protocol ReaderFollowedInterestsService: AnyObject {
+    /// Fetches the users locally followed interests
+    /// - Parameter completion: Called after a fetch, will return nil if the user has no interests or an error occurred
+    func fetchFollowedInterestsLocally(completion: @escaping ([ReaderTagTopic]?) -> Void)
+
+    /// Fetches the users followed interests from the network, then returns the sync'd interests
+    /// - Parameter completion: Called after a fetch, will return nil if the user has no interests or an error occurred
+    func fetchFollowedInterestsRemotely(completion: @escaping ([ReaderTagTopic]?) -> Void)
+}
+
+// MARK: - CoreData Fetching
+extension ReaderTopicService: ReaderFollowedInterestsService {
+    public func fetchFollowedInterestsLocally(completion: @escaping ([ReaderTagTopic]?) -> Void) {
+        completion(followedInterests())
+    }
+
+    public func fetchFollowedInterestsRemotely(completion: @escaping ([ReaderTagTopic]?) -> Void) {
+        fetchReaderMenu(success: { [weak self] in
+            guard let `self` = self else {
+                completion(nil)
+                return
+            }
+
+            self.fetchFollowedInterestsLocally(completion: completion)
+        }) { (error) in
```

Comment:
Tiny nit picking, you don't have to use parentheses when you have only one parameter in the block 
`{ error in
}`